content/en/blog/contributor-contest/index.md

```diff
@@ -0,0 +1,128 @@
++++
+title = 'Win tickets to "KubeCon + CloudNativeCon & Open source summit, Shanghai, China 2019"'
+subtitle = "KubeEdge is an open source project, recently sandboxed under CNCF. It is one of the few, if not the only, open source edge computing platforms that has both cloud and edge parts opensourced."
+date = 2019-04-22
+lastmod = 2019-04-22
+
+draft = false
+
+# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
+authors = ["KubeEdge"]
+
+# Tags and categories
+# For example, use `tags = []` for no tags, or the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
+tags = ["KubeEdge", "kubeedge", "edge computing", "kubernetes edge computing", "K8S edge orchestration", "edge computing platform", "KubeCon", "cloud native"]
+categories = ["General", "Announcements"]
+summary = "KubeEdge is an open source project, recently sandboxed under CNCF. It is one of the few, if not the only, open source edge computing platforms that has both cloud and edge parts opensourced."
+
+# Projects (optional).
+#   Associate this post with one or more of your projects.
+#   Simply enter your project's folder or file name without extension.
+#   E.g. `projects = ["deep-learning"]` references 
+#   `content/project/deep-learning/index.md`.
+#   Otherwise, set `projects = []`.
+# projects = ["internal-project"]
+
+# Featured image
+# To use, add an image named `featured.jpg/png` to your page's folder. 
+[image]
+  # Caption (optional)
+  caption = ""
+
+  # Focal point (optional)
+  # Options: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight
+  #focal_point = "Center"
++++
+
+  
+
+Any open source project is successful only if the community keeps contributing regularly. We want to encourage our community members and 
+also anyone who wants to start contributing to KubeEdge. To sweeten the deal we have created a contest for you. To put it in simple words,
```

Comment:
KubeEdge is a CNCF Sandbox project that extends K8s from Cloud to Edge. We would like to invite you to join us in furthering this project and making it useable for everyone. To make this contribution effort more fun, we’re proposing a contribution competition. See below for details. May the best contributor win!